dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22AppConfigPop.war/resources/WEB-INF/web.xml

```diff
@@ -13,28 +13,23 @@
     xmlns="http://xmlns.jcp.org/xml/ns/javaee" 
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
     xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd">
-    
-  <display-name>AppConfigPop</display-name>
-  
-  <context-param>
-    <param-name>javax.faces.STATE_SAVING_METHOD</param-name>
-    <param-value>server</param-value>
-  </context-param>
- 
-  <servlet>
-    <servlet-name>Faces Servlet</servlet-name>
-    <servlet-class>javax.faces.webapp.FacesServlet</servlet-class>
-  </servlet>
-  
-  <welcome-file-list>
-    <welcome-file>index.jsf</welcome-file>
-  </welcome-file-list>
```

Comment:
Nothing really important, is it worth updating the copyright date because of this removal?

Reply:
I didn't update it b/c I didn't feel it was significant enough just removing that minor welcome-file-list as some minor cleanup similar to updating the formatting. It isn't used since there is no index.jsf and was likely just a copy paste mistake!